02-peer-protocol.md

```diff
@@ -1465,6 +1465,9 @@ The receiver of `stfu`:
     - SHOULD NOT send any more update messages.
     - MUST reply with `stfu` once it can do so.
 
+Both nodes:
+  - MUST disconnect after 60 seconds of quiescence if the HTLCs are pending.
```

Comment:
I know this has already been merged but I do not believe this is a `MUST`. This is up to node policy and can be initiated by either side. Especially considering that time is both relative and difficult in distributed systems this ought to be a `SHOULD`

Reply:
What is "pending" here? I believe it is a different definition than the one above.